doc-content/enterprise-only/openshift/rhdm76-authoring-ha-ref.adoc

```diff
@@ -329,6 +332,7 @@ Http Get on http://localhost:8080/services/rest/server/healthcheck
 |`APPFORMER_JMS_BROKER_PORT` | -- | 61616
 |`APPFORMER_JMS_BROKER_USER` | The user name to connect to the JMS broker. | `${APPFORMER_JMS_BROKER_USER}`
 |`APPFORMER_JMS_BROKER_PASSWORD` | The password to connect to the JMS broker. | `${APPFORMER_JMS_BROKER_PASSWORD}`
+|`JAVA_MAX_MEM_RATIO` | Decision Central Container JVM max memory ratio. -Xmx is set to a ratio of the container available memory. The default is 80 which means 80% of the available memory is used as an upper boundary. You can skip this mechanism by setting this value to 0 in which case no -Xmx option is added. | `${DECISION_CENTRAL_JAVA_MAX_MEM_RATIO}`
```

Comment:
The default is 80, which means the upper boundary is 80% of the available memory. To skip adding the -Xmx option, set this value to 0. | `${DECISION_CENTRAL_JAVA_MAX_MEM_RATIO}`

Reply:
This file is autogenerated from the template, and I never got to edit the template as this was a last-moment addition. I can make the change but it will not survive the next autogeneration.

Reply:
Ok. Please open a Jira to request an edit to the template so the autogeneration is fixed for next time.